run.sh

```diff
@@ -0,0 +1,21 @@
+# Set & move to home directory
+source set_env.sh
+cd "$SNORKELHOME"
+
+# Make sure the submodules are installed
+git submodule update --init --recursive
```

Comment:
Upon closer inspection this submodule is for the treedlib repo. I don't think there should be a version issue because on the master branch they specify a commit hash in their submodule.